accelerator/config.c

```diff
@@ -123,7 +123,7 @@ status_t ta_core_default_init(ta_config_t* const ta_conf, iota_config_t* const i
   ta_conf->host = TA_HOST;
   ta_conf->port = TA_PORT;
   ta_conf->thread_count = TA_THREAD_COUNT;
-#ifdef ENABLE_MQTT
+#ifdef MQTT_ENABLE 
```

Comment:
Can we enforce consistent naming scheme for such macros? `ENABLE_MQTT` or `MQTT_ENABLE`?

Reply:
@HowJMay, Let's clarify the name scheme.

Reply:
Got it